docs/source/get_started/new_project.md

```diff
@@ -112,7 +112,9 @@ The nodes are stored in `src/get_started/nodes.py`:
 
 ### Visualise the project
 
-This is a swift introduction to show how to visualise the project with Kedro-Viz. See the [visualisation documentation](../visualisation/kedro-viz_visualisation) for more detail.
+This is a swift introduction to show how to visualise the project with Kedro-Viz. See the [visualisation documentation](../visualisation/kedro-viz_visualisation) for more detail. 
+
+Kedro-Viz package needs to be installed seperately as it is not part of standard Kedro installation.
```

Comment:
```suggestion
The Kedro-Viz package needs to be installed separately as it is not part of the standard Kedro installation:

```bash
pip install kedro-viz
``
```
The last line should have 3 backticks, but I can't write a suggestion block with that.

Reply:
@stichbury Curious, do you prefer "as it is not part of the standard Kedro installation", "as it is a plugin", or something else?

Reply:
@deepyaman Thanks for your review. Please let me know whether understanding below is correct and which direction you would prefer

Direction 1:

I will wait for your and stichbury's decision on sentence per you latest question and send another PR with added sign-off for review

Direction 2:
 I don't need to do anything more and this will be updated from your side and merged

Direction 3：
The new PR mentioned in direction 1 can also include the update for https://kedro.readthedocs.io/en/stable/visualisation/kedro-viz_visualisation.html and it should follow exactly the same change made for this PR (docs/source/get_started/new_project.md)

Reply:
> I will wait for your and stichbury's decision on sentence per you latest question and send another PR with added sign-off for review

You can just update this PR by pushing additional commits to your branch (e.g. to modify that other file). Up to you if you want to wait for @stichbury's comments before making the changes; I just defer to her on some of this because her writing knowledge is much better. :)

Reply:
Sure, a new commit is now pushed to update the change per your previous suggestion and also for https://kedro.readthedocs.io/en/stable/visualisation/kedro-viz_visualisation.html

Sign-off was added in the second commit but it seems like the DCO still shows unsuccessful

Reply:
Thank you @deepyaman and @grhaonan! I appreciate the fixes and I am happy to approve this and merge it when it's signed off. Don't worry about the DCO -- I can resolve that by overriding it.

I like "The Kedro-Viz package needs to be installed separately as it is not part of the standard Kedro installation" I think. It still does sort of leave you wondering "why?" but it's OK and the word "package" kind of gives it away. I wondered about "Kedro-Viz needs to be installed as it is a separate package to Kedro" but that's a bit clunky. Let's leave it with what you have.